website/blog/2020-03-24-version-0.62.md

```diff
@@ -0,0 +1,94 @@
+
+Going forward, Apple TV support for React Native will be maintained in [react-native-community/react-native-tvos](https://github.com/react-native-community/react-native-tvos) along with the corresponding `react-native-tvos` NPM package. This is a full fork of the main repository, with only the changes needed to support Apple TV.
+
+Releases of `react-native-tvos` will be based on the public release of React Native. For this 0.62 release of `react-native` please upgrade Apple TV projects to use `react-native-tvos` 0.62.
+
+## Even more upgrading support
+
+When 0.61 was released, the community introduced a new tool called [upgrade-helper](https://react-native-community.github.io/upgrade-helper/) to better support developers in their upgrading effort. It has been a great addition and we are grateful for all the positive vibes that we felt around that project.
+
+That said, given that upgrading can still be an hard task because of the inevitable evolution of the native platforms, we have decided to one-up the dedicated upgrading resources: we are eager to announce [Upgrade-Support](https://github.com/react-native-community/upgrade-support), a central place to request and give help when upgrading your app.
+
+Upgrade Support is a GitHub repo hosting an issue tracker where people can post any issues they have while upgrading their projects. This should help increase the likelihood that people can find others with the same problems and work together to figure out a solution. For more information around this project, please refer to its [README](https://github.com/react-native-community/upgrade-support).
+
+## Other Improvements
+
+- **LogBox**: We’re adding the new LogBox error and warning experience as an opt-in; to enable it, add `require('react-native').unstable_enableLogBox()` to your `index.js` file.
+- **React DevTools v4**: This change includes an upgrade to the [latest React DevTools](https://reactjs.org/blog/2019/08/15/new-react-devtools.html) which offers significant performance gains, an improved navigation experience, and full support for React Hooks.
+- **Accessibility improvements**: We’ve made improvements to accessibility including adding [accessibilityValue](https://reactnative.dev/docs/accessibility#accessibilityvalue-ios-android), missing props on [Touchables](https://github.com/facebook/react-native/commit/8c0c860e38f57e18296f689e47dfb4a54088c260), `onSlidingComplete` [accessibility events](https://github.com/facebook/react-native/commit/c7aa6dc8270c0eabc913fe6c617c8131e3f4b3c5), and changing the default role of Switch component from ‘button’ to ‘switch’.
+
+## Breaking Changes
+
+- **Remove PropTypes**: We're removing `propTypes` from core components in order to release the app size impact of React Native core and to favor static type systems which check at compile time instead of runtime.
```

Comment:
```suggestion
- **Remove PropTypes**: We're removing `propTypes` from core components in order to reduce the app size impact of React Native core and to favor static type systems which check at compile time instead of runtime.
```